app/Http/Requests/User/UserSettingsRequest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Requests\User;
+
+use App\Http\Requests\AbstractEmptyRequest;
+use App\Models\User;
+use App\Policies\UserPolicy;
+use Illuminate\Support\Facades\Gate;
+
+class UserSettingsRequest extends AbstractEmptyRequest
```

Comment:
I am unhappy with the name of this class.

 1. All other request classes start with a verb. For consistency, this class should do so, too.
 2. More importantly, we already have the class `SetUserSettingsRequest`. Now, we have `UserSettingsRequest` on top. I haven't yet check all the details, but it appears that you have split the previous one request into two. This is fine, but at least the name of both classes should be chosen in a way that their difference and their different purpose is clear from the name. Having two request classes with nearly identical names is not good. Right now, I have no suggestions, but maybe I will come up with two names later after I have read all the code and know which class is used where.

Reply:
Removed.